src/format.ts

```diff
@@ -110,6 +112,11 @@ export function formatMarkdown(milestone: string, changes: ChangeEntry[]): strin
 	add("Fixes", (cs) => collectChanges(cs, TYPE_FIX))
 	add("Chore", (cs) => collectChanges(cs, TYPE_CHORE))
 
+	// Add the last line with the URL of MAJ.MIN changelog.
+	body.push({
+		p: `See [the CHANGELOG ${milestoneVersion.toMinor()}](../main/CHANGELOG/CHANGELOG-${milestoneVersion.toMinor()}.md) for more details.`,
+	})
```

Comment:
This action is only responsible for generating text, not placing it in a repo file structure. Why should it know about paths the repo? 

All the paths decisions are made in CI of the target repo. Let's do it in the [milestone-changelog](https://github.com/deckhouse/deckhouse/blob/main/.github/actions/milestone-changelog/action.yml#L61) in d/d, where we concat files with bash and choose where to place files.